### PR TITLE
added ubuntu 16.04 image of indy-node

### PR DESCRIPTION
**Pull request by udosson**
This PR contains the Dockerfile and GHA to build & publish a Docker image of Indy-Node that is based on Ubuntu 16.04.

Thanks to @Echsecutor and @mgmgwi for their contribution. This PR just copies and adjusted existing stuff.

Signed-off-by: udosson <r.klemens@yahoo.de>

### CODE REVIEW

**Comment by udosson** [comment]
The Makefile needs to be adjusted as well in a future commit.

File: .github/workflows/build-ubuntu16.yml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        run: echo "::set-output name=lowercase::$(echo ${{ github.repository_owner }} | tr \"[:upper:]\" \"[:lower:]\")"
+        shell: bash
+
+      - name: Meta for base
+        id: meta_base
+        uses: docker/metadata-action@v3
+        with:
+          images: ghcr.io/${{ steps.repo_owner.outputs.lowercase }}/indy-node-container/base
+          tags: |
+            type=raw,value=ubuntu
```
Comment (Echsecutor) [inline .github/workflows/build-ubuntu16.yml:41]: is meta for base used at all?

File: .github/workflows/build-ubuntu16.yml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+      - name: Meta for indy_node
+        id: meta_indy_node
+        uses: docker/metadata-action@v3
+        with:
+          images: ghcr.io/${{ steps.repo_owner.outputs.lowercase }}/indy-node-container/indy_node
+          # Note: no latest tag for the ubuntu build
+          flavor: |
+            suffix=-ubuntu
```
Comment (Echsecutor) [inline .github/workflows/build-ubuntu16.yml:50]: ubuntu16
and similar change to ubuntu18 in that build action

File: .github/workflows/trivy-ubuntu16.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    name: Trivy Analysis (indy_node:ubuntu16)
+    runs-on: "ubuntu-18.04"
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Run Trivy on indy_node:ubuntu16
+        uses: aquasecurity/trivy-action@master
+        with:
+          image-ref: 'ghcr.io/idunion/indy-node-container/indy_node:ubuntu'
```
Comment (Echsecutor) [inline .github/workflows/trivy-ubuntu16.yml:20]: ubuntu16

**Review by Echsecutor — CHANGES_REQUESTED** [review]

File: .github/workflows/build-ubuntu16.yml (hunk 1 above)
Comment (udosson) [reply .github/workflows/build-ubuntu16.yml]: No. Removed metadata for base image in the recent commit

File: .github/workflows/build-ubuntu16.yml (hunk 2 above)
Comment (udosson) [reply .github/workflows/build-ubuntu16.yml]: done

File: .github/workflows/trivy-ubuntu16.yml (hunk 3 above)
Comment (udosson) [reply .github/workflows/trivy-ubuntu16.yml]: done and for `/trivy-ubuntu19.yml` as well

**Review by Echsecutor — APPROVED** [review]

**Pull request merged**